pkg/auth/keymanager.go

```diff
@@ -140,11 +138,11 @@ func (km *KeyManager) fetchKeys(keysEndpointURL string) ([]*PublicKey, error) {
 	defer func() {
 		_, err := ioutil.ReadAll(res.Body)
 		if err != io.EOF && err != nil {
-			km.logger.Println("failed read remaining data before closing response")
+			log.Info(nil, "failed read remaining data before closing response")
 		}
 		err = res.Body.Close()
 		if err != nil {
-			km.logger.Println("failed to close response after reading")
+			log.Info(nil, "failed to close response after reading")
```

Comment:
```suggestion
			log.Error(nil, err, "failed to close response after reading")
```

Reply:
Since we used Println, i went with info, but i didn't thoroughly review. Will change this..

Reply:
I think Alexey's point is that it's good to include the error object in the log :)

Reply:
when logging with info there is no option to pass error and since println seems more like something that aligns with info, i had used info instead of error without throoughlylooking at the code. but changing it to log.Error (or now log.Errorf) allows you to pass the error :) fixed here: 00bf17d961e783ddd6ad166872fdd5903f82a7c1